rc/base/autowrap.kak

```diff
@@ -37,9 +39,9 @@ def -hidden autowrap-cursor %{ eval -save-regs '/"|^@m' %{
 } }
 
 def autowrap-enable -docstring "Wrap the lines in which characters are inserted" %{
-    hook -group autowrap window InsertChar [^\n] autowrap-cursor
+    hook -group autowrap buffer InsertChar [^\n] autowrap-cursor
```

Comment:
I wonder if we even need that autowrap-enable command, we could just let users put that hook themselves, on the scope they prefer.

But if we provide a command, I'd rather have it put hook on the current window scope, rather than on the buffer, as I'd rather leave the user choose (he can just always do for all windows if he wants to).